src/components/Profile/Welcome.vue

```diff
@@ -0,0 +1,40 @@
+<template>
+  <div class="welcome">
+    <img :src="familyImg"/>
+    <div>
+      <span class="title">Welcome, the {{ name }} Family!</span>
+      <p>Lorem ipsum...</p>
+    </div>
+  </div>
+</template>
+
+<script>
+export default {
```

Comment:
Look up functional components vue and see if you want to make this one.

Reply:
I changed Welcome to be functional, not sure if I did it exactly how you were imagining because it only involved changing a couple lines

Reply:
Yup this is perfect. It just boosts the performance of the application since it doesn't need to watch data.